pages/user guides/discover_nmap.md

```diff
@@ -8,70 +8,61 @@ permalink: discover-nmap.html
 ---
```

Comment:
L7 above, should the link here match the title? perhaps, `host-discovery.html`?

Reply:
yes, yes. 👍